packages/notebook/src/panel.ts

```diff
@@ -99,6 +101,16 @@ export class NotebookPanel extends DocumentWidget<Notebook, INotebookModel> {
         }
       });
     }
+
+    const model = sender.contentsModel;
+    if (
+      state === 'completed' &&
+      model &&
+      !model.renamed == true &&
+      model.name.startsWith('Untitled')
```

Comment:
Just wondering if this will be robust enough, or wether we should take this from the place it is defined? (it is an option that is configurable, right?)

The `Untitled` I mean

Reply:
it might not be robust code-wise, but I thought, because the Name File popup is to avoid having the `Untitled`s (and I am assuming if folks are changing `Untitled` to something else, they might want to keep it, instead of renaming). So I am hoping using `Untitled` would be sufficient for now?

Reply:
Yeah it should be good for now

Reply:
Looks like this is configurable with `ContentsManager.untitled_file`:

https://jupyter-server.readthedocs.io/en/latest/full-config.html?#options